dll/win32/shell32/wine/pidl.c

```diff
@@ -1682,12 +1682,24 @@ LPITEMIDLIST _ILCreateBitBucket(void)
     return _ILCreateGuid(PT_GUID, &CLSID_RecycleBin);
 }
 
+LPITEMIDLIST _ILCreateFonts(void)
```

Comment:
If those two new functions are _not_ in Wine's shell32 as well, they should certainly be surrounded inside a `#ifdef __REACTOS__  .... #endif` block.